addons/website_slides/static/src/xml/slide_quiz_create.xml

```diff
@@ -0,0 +1,97 @@
+                    </button>
+                </div>
+            </form>
+            <t t-if="widget.update" t-call="slide.quiz.update.buttons"/>
+            <t t-else="" t-call="slide.quiz.create.buttons"/>
+        </div>
+    </t>
+
+    <t t-name="slide.quiz.answer.line">
+        <div class="o_wslides_js_quiz_answer row align-items-center mb-1" t-attf-data-answer-id="#{answer ? answer.id : ''}" >
+            <a class="o_wslides_js_quiz_is_correct col-1 text-center" title="Select one good answer">
+                <label class="my-0">
+                    <input t-if="answer and answer.is_correct" class="d-none" type="radio" name="radio" checked="true" />
+                    <input t-else="" class="d-none" type="radio" name="radio" />
+                    <i class="fa fa-lg fa-check-circle-o" />
+                </label>
+            </a>
+            <input type="text" class="form-control col-9" placeholder="Enter your answer" t-attf-value="#{answer ? answer.text_value : ''}"/>
+            <i class="o_wslides_js_quiz_add_answer fa fa-lg fa-plus-circle p-2" title="Add another answer" />
+            <i class="o_wslides_js_quiz_remove_answer fa fa-lg fa-trash-o p-2" title="Remove this answer" />
+        </div>
+    </t>
+
+    <t t-name="slide.quiz.create.buttons">
+        <div class="d-flex">
+            <a class="o_wslides_js_quiz_validate_question btn btn-primary text-white border" role="button">
+                <span class="d-none d-sm-inline-block">Save &amp; Close</span>
+            </a>
+            <a class="o_wslides_js_quiz_validate_question o_wslides_js_quiz_create_next btn btn-primary text-white border ml-2" role="button">
+                <span class="d-none d-sm-inline-block">Save &amp; New</span>
+            </a>
+            <a class="o_wslides_js_quiz_cancel_question btn btn-light border ml-2" role="button">
+                <span class="d-none d-sm-inline-block">Cancel</span>
+            </a>
+        </div>
+    </t>
+
+    <t t-name="slide.quiz.update.buttons">
+        <a class="o_wslides_js_quiz_validate_question o_wslides_js_quiz_update btn btn-primary text-white border" role="button">
+            <span class="d-none d-sm-inline-block">Update</span>
+        </a>
+        <a class="o_wslides_js_quiz_cancel_question btn btn-light border ml-2" role="button">
+            <span class="d-none d-sm-inline-block">Cancel</span>
+        </a>
+    </t>
+
+    <t t-name="slide.quiz.new.quiz.button">
+        <a class="o_wslides_js_quiz_add btn btn-light border ml-2" role="button">
+            <i class="fa fa-plus mr-2"/>
+            <span class="d-none d-sm-inline-block">Add Quiz</span>
+        </a>
+    </t>
+
+    <t t-name="slide.quiz.new.question.button">
+        <a class="o_wslides_js_quiz_add btn btn-light border ml-2" role="button">
+            <i class="fa fa-plus mr-2"/>
+            <span class="d-none d-sm-inline-block">Add Question</span>
+        </a>
+    </t>
```

Comment:
I know this is a work in progress and you might already know, but I don't think these 2 templates are still used.